Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM alpine:3.8 as runner
+FROM alpine:latest
+
+ENTRYPOINT ["/usr/sbin/bioscfg"]
```

Comment:
Any reason why we moved the `entrypoint` up here? I don't know that it will affect the execution, but I've never really seen it NOT be the last line of Dockerfile.

Reply:
no, ill move it back. I forgot to change it back. I was dealing with a weird bug and trying some wild stuff to see what was up.